tests/e2e/specs/stories-editor/cta.test.js

```diff
@@ -47,9 +48,7 @@ describe( 'Stories Editor Screen', () => {
 		expect( nodes ).toHaveLength( 1 );
 	} );
 
-	// Disable reason: Test is flaky in headless mode - needs to be investigated.
-	// eslint-disable-next-line jest/no-disabled-tests
-	it.skip( 'should display validation error when CTA block is on the first Page', async () => {
+	it( 'should display validation error when CTA block is on the first Page', async () => {
 		await createNewPost( { postType: 'amp_story' } );
 
 		const firstPageClientId = ( await getAllBlocks() )[ 0 ].clientId;
```

Comment:
We have lots of instances of `( await getAllBlocks() )[ 0 ].clientId`. I feel like we could use a `getPageClientId( pageNum )` utility function for that. And perhaps even a `getCurrentPageClient()` helper.